layer1/P.cpp

```diff
@@ -643,7 +643,7 @@ static int getExplicitValence(ObjectMolecule const* obj, size_t atm)
  *   RDKit::Atom::getDegree()
  *   OBAtom::GetExplicitDegree() [Open Babel 3.0]
  */
-static unsigned getExplicitDegree(ObjectMolecule const* obj, size_t atm)
+unsigned getExplicitDegree(ObjectMolecule const* obj, size_t atm)
```

Comment:
I probably should have mentioned in the previous review, but this might be better served in the `ObjectMolecule` or `AtomInfo` file. (But I'll take care of that later).

Reply:
I can try to move to `AtomInfo`;